### PR TITLE
Remove DBSecurityGroup CRD from rds-controller

### PR DESCRIPTION
**Pull request by brucegucode**
Issue #, if available: [issue-1290](https://github.com/aws-controllers-k8s/community/issues/1290)

Description of changes:

Since now all new RDS db instances are VPC instances, there is no need to still leave CRD DBSecurityGroup open. Now RDS db instances are using VPC security group. 
(According to https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBSecurityGroup.html )

Remove DBSecurityGroup CRD from rds-controller. Also remove the db_security_group e2e test. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: generator.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ ignore:
     #- DBParameterGroup
     - DBProxy
     - DBProxyEndpoint
-    #- DBSecurityGroup
+    - DBSecurityGroup
```
[the remark this answers is not in the document]
Comment (brucegucode) [reply generator.yaml:14]: ack, will do. I'm still trying to figure out why the code generator still generate `f26iter ` which is not used and cause test failure. I will address it in next revision in this pr.

[the remark this answers is not in the document]
Comment (jaypipes) [reply generator.yaml:14]: @brucegucode you will still want to ignore the DBInstance.DBSecurityGroups field path as well.

**Review by RedbackThomson — CHANGES_REQUESTED** [review]
Please remove the tests for this resource as well. Deleting:
- https://github.com/aws-controllers-k8s/rds-controller/blob/main/test/e2e/tests/test_db_subnet_group.py
- https://github.com/aws-controllers-k8s/rds-controller/blob/main/test/e2e/db_subnet_group.py
- https://github.com/aws-controllers-k8s/rds-controller/blob/main/test/e2e/resources/db_subnet_group_2az.yaml

Hopefully that does the trick

**Comment by RedbackThomson** [comment]
Great! Glad we figured that one out.
/approve

**Pull request merged**